### ISSUE TITLE
use go mod

### ISSUE THREAD

**Issue by joeblew99** [andrewkroh/gvm#16]
https://github.com/Masterminds/glide#go-modules

:)

**Comment by joeblew99** [comment]
also glide up is failing in the same way that using GO111MODULE=on is.


```
glide up
[INFO]  Downloading dependencies. Please wait...
[INFO]  --> Fetching github.com/hashicorp/go-version
[INFO]  --> Fetching github.com/pkg/errors
[INFO]  --> Fetching updates for gopkg.in/alecthomas/kingpin.v2
[INFO]  --> Fetching updates for github.com/Sirupsen/logrus
[ERROR] Update failed for github.com/Sirupsen/logrus: The Remote does not match the VCS endpoint
[ERROR] Failed to do initial checkout of config: The Remote does not match the VCS endpoint
```

```
/Users/apple/workspace/go/src/github.com/andrewkroh/gvm
x-MacBook-Pro:gvm apple$ cd cmd
x-MacBook-Pro:cmd apple$ ls
gvm
x-MacBook-Pro:cmd apple$ cd gvm
x-MacBook-Pro:gvm apple$ GO111MODULE=on go build gvm.go
go: github.com/Sirupsen/logrus@v1.4.2: parsing go.mod: unexpected module path "github.com/sirupsen/logrus"
go: error loading module requirements
```

**Comment by joeblew99** [comment]
fixed 
just do a find replace of “Sirupsen” to “sirupsen”

https://github.com/nicholasjackson/minke/issues/56

**Comment by joeblew99** [comment]
final go mod


module github.com/andrewkroh/gvm

go 1.12

require (
	github.com/alecthomas/template v0.0.0-20160405071501-a0175ee3bccc // indirect
	github.com/alecthomas/units v0.0.0-20151022065526-2efee857e7cf // indirect
	github.com/hashicorp/go-version v1.2.0
	github.com/pkg/errors v0.8.1
	github.com/sirupsen/logrus v1.4.2
	gopkg.in/alecthomas/kingpin.v2 v2.2.6 // indirect

)

**Comment by andrewkroh** [comment]
I have opened #17 to update the repo.

**Comment by joeblew99** [comment]
thanks !

**Comment by andrewkroh** [comment]
This has been released as v0.2.1.

**Comment by joeblew99** [comment]
thanks for quick turnaround @andrewkroh  !!
